Sources/GRPCCodeGen/Internal/Translator/IDLToStructuredSwiftTranslator.swift

```diff
@@ -49,6 +49,40 @@ struct IDLToStructuredSwiftTranslator: Translator {
   }
 }
 
+extension IDLToStructuredSwiftTranslator {
+  private func translateImport(
+    dependency: CodeGenerationRequest.Dependency
+  ) throws -> ImportDescription {
+    var importDescription = ImportDescription(moduleName: dependency.module)
+    if let item = dependency.item {
+      if let matchedKind = ImportDescription.Kind.allCases.first(where: {
+        "\($0)" == item.kind.value.description
+      }) {
+        importDescription.item = ImportDescription.Item(kind: matchedKind, name: item.name)
+      } else {
+        throw CodeGenError(
+          code: .invalidKind,
+          message: "Invalid kind name for import: \(item.kind.value.description)"
+        )
+      }
+    }
+    if let spi = dependency.spi {
+      importDescription.spi = spi
+    }
+
+    let preconcurrency = dependency.preconcurrency.value
+    switch preconcurrency {
```

Comment:
Nit, but as you're not using `preconcurrency` anywhere else, you can switch directly over the value without declaring the let before:
```suggestion
    switch dependency.preconcurrency.value {
```